app/public/dist/client/locales/en/translation.json

```diff
@@ -1256,383 +1256,383 @@
-    "HYDRO_PUMP": "Deal [20,40,80,SP] SPECIAL to all enemy Pokémon behind the target",
-    "DRACO_METEOR": "Deal [50,SP] SPECIAL to all enemy Pokémon",
+    "KING_SHIELD": "PROTECT the user for 1.5 seconds, gain [10,20,30,SP] SHIELD and swap positions with the farthest enemy Pokémon.",
+    "U_TURN": "Gain [15,30,50,SP] SHIELD, swap positions with the target and inflict them with CHARM for 1 second.",
+    "EXPLOSION": "Deal [50,100,200,SP] SPECIAL to all adjacent enemy Pokémon. Damages the user.",
+    "NIGHTMARE": "SILENCE all enemy Pokémon for [1.5,3,6,ND=1] seconds. Any Pokémon already suffering from SLEEP, FLINCH or SILENCE take [40,80,150,SP] SPECIAL.",
+    "CLANGOROUS_SOUL": "Buff the user's and adjacent allied Pokémon's ATK, DEF and SPE_DEF by [1,2,4,SP].",
+    "BONEMERANG": "Throw a bone like a boomerang through the enemy team, dealing [30,60,90,SP] SPECIAL to every enemy Pokémon in its path.",
+    "GROWL": "Cause adjacent enemy Pokémon to FLINCH for 3 seconds and reduce their ATK by [3,5,7,SP].",
+    "RELIC_SONG": "Emit 3 notes. The first two notes grant [10,SP] SHIELD to all allied Pokémon. The third note puts all enemy Pokémon to SLEEP for [2,SP=0.5] seconds.",
+    "FAIRY_WIND": "Restore [5,10,20,SP] PP to all allied Pokémon other than the user.",
+    "DISARMING_VOICE": "Inflict CHARM onto all enemy Pokémon within a 2 tile radius for [1,SP] seconds.",
+    "HIGH_JUMP_KICK": "Deal [50,100,200,SP] SPECIAL and steal up to 70 PP from the target.",
+    "GRASS_WHISTLE": "Put [1,2,4] enemy Pokémon to SLEEP for 2 seconds.",
+    "TRI_ATTACK": "Deal [30,60,SP] SPECIAL. May either inflict BURN, FREEZE or PARALYSIS onto the target.",
+    "ECHO": "Deal [3,6,9,SP] SPECIAL to all enemy Pokémon. Every time the user uses this ability, the damage increases by [3,6,9,SP].",
+    "PETAL_DANCE": "Deal [20,30,50,SP] SPECIAL to the [3,4,5] closest enemy Pokémon.",
+    "HYPER_VOICE": "Deal [30,60,150,SP] SPECIAL to all enemy Pokémon in the same row as the target and inflict CONFUSION for [1,2,3] seconds.",
+    "SHADOW_CLONE": "Summon a clone with [50,SP]% of the user's max HP and give it one of the user's held items.",
+    "VOLT_SWITCH": "Dash into the enemy team's back line and deal [30,60,120,SP] SPECIAL to all enemy Pokémon in the path.",
+    "NUZZLE": "Dash to the farthest opponent, deal [30,60,120,SP] SPECIAL and inflict it with PARALYSIS for [3,4,5] seconds.",
+    "LIQUIDATION": "Deal [20,40,80,SP] SPECIAL and lower the target's DEF by [2,4,8,SP].",
+    "BURN": "BURN all enemy Pokémon for [5,10,20] seconds.",
+    "POISON": "POISONNED the [1,2,3] closest enemy Pokémon for [3,6,9,SP=0.5] seconds.",
+    "SLEEP": "Put [1,2,3] enemy Pokémon in the back line to SLEEP for [2,SP] seconds.",
+    "PROTECT": "PROTECT the user, becoming invulnerable for [1,3,5,SP=0.5] seconds.",
+    "BLIZZARD": "FREEZE all enemy Pokémon for 2 seconds and deal [5,10,15,SP] SPECIAL, doubled if the target already suffers from FREEZE.",
```

Comment:
not clear if the attack is doubled or only the damage. This makes a big difference against fighting for example. That's why we clarify it as Double damage